src/leap/mx/tcp_map.py

```diff
@@ -27,11 +30,14 @@
 TCP_MAP_CODE_PERMANENT_FAILURE = 500
 
 
-class LEAPPostfixTCPMapServerFactory(ServerFactory):
+class LEAPPostfixTCPMapServerFactory(ServerFactory, object):
```

Comment:
If you don't import object, python's `abc.ABCMeta` will complain with a strange error:

```
>>> import tcp_map
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "tcp_map.py", line 33, in <module>
    class LEAPPostfixTCPMapServerFactory(ServerFactory):
  File "/usr/lib/python2.7/abc.py", line 87, in __new__
    cls = super(ABCMeta, mcls).__new__(mcls, name, bases, namespace)
TypeError: Error when calling the metaclass bases
    a new-style class can't have only classic bases
```

This has to do with the way python implements subclassing and the relation between old and new style of classes. I also don't know the internals, but I found this in some foruns. Will add a note on the code so we know it was not added by accident.